### ISSUE TITLE
🚨 🚨 🚨 🚨 THIS REPO IS NOT MAINTAINED 🚨 🚨 🚨 🚨

### ISSUE THREAD

**Issue by ghost** [cloudverve/wp-sync-db#6]
I just came form Here: https://github.com/wp-sync-db/wp-sync-db/issues/167 and it looks like I've ended up at another graveyard.

**Comment by NoWorries** [comment]
not maintained, but still working (for now)

**Comment by ghost** [comment]
The Maintainers website does not seem to work, so I am assuming they have gone out of business and left this repo behind. As you already implied, it is only a matter of time for this to stop working.

I am considering to fork this and maintain my own version but since I am not a very good programmer I probably shouldn't. Then again it still might be better than nothing. It really depends on how well I can understand the code.

**Comment by ghost** [comment]
@notslang I found you in a comment in [compatibility/wp-sync-db-compatibility.php](https://github.com/cloudverve/wp-sync-db/blob/master/compatibility/wp-sync-db-compatibility.php), do you know anything about what happened, would you even be willing to maintain this?

**Comment by notslang** [comment]
@jayhybrid - Sorry, I haven't worked with PHP or WordPress in several years. Nowadays I'm doing Elixir work. I haven't found anyone willing to maintain the repo either. See: https://github.com/wp-sync-db/wp-sync-db#looking-for-a-maintainer

**Comment by trobriander** [comment]
@notslang Just so I'm clear: it's legal to fork the original code from DB Pro, and remove all of the license-checking code?

**Comment by notslang** [comment]
@trobriander Yep, it's licensed under the GPL. You just need to remove the branding, because that's trademarked.
